modules/context/access_log.go

```diff
@@ -20,20 +22,51 @@ type routerLoggerOptions struct {
 	Start          *time.Time
 	ResponseWriter http.ResponseWriter
 	Ctx            map[string]interface{}
+	RequestID      *string
 }
 
 var signedUserNameStringPointerKey interface{} = "signedUserNameStringPointerKey"
 
+const keyOfRequestIDInTemplate = ".RequestID"
+
+// According to:
+// In OpenTracing and OpenTelemetry, the maximum length of trace id is 256 bits (32 bytes).
+// MD5 output is 16 or 32 bytes.
+// UUID output is 36 bytes (including four ‘-’)
+// SHA1 output is 40 bytes
+// So, we accept a Request ID with a maximum character length of 40
+const maxRequestIDBtyeLength = 40
```

Comment:
Typo:
```suggestion
const maxRequestIDByteLength = 40
```

Or maxRequestIDLength is good enough, we know it's bytes not bits.